### PR TITLE
Mobile step template

### PR DESCRIPTION
**Pull request by youngfreezyVA**
resolves  #14047
https://github.com/department-of-veterans-affairs/va.gov-team/issues/14047

also fixes issues with ghost alerts, as i was seeing a couple when working on this issue.

Ghost alerts description:
https://github.com/department-of-veterans-affairs/va.gov-team/blob/6f340c1808fed0970458be5e436893715e71362c/products/content/tier-2-content-IA-and-design/learning-center-mvp/cms/readme.md#notes

a note:
-I think we need a mobile friendly version of alerts, if we don't already have one. couldn't find any mocks or anything on github / in the codebase related to this. I would suggest grooming a mobile alerts story next

Here is what it looks like now ( http://localhost:3001/preview?nodeId=6899 ) :

https://drive.google.com/file/d/1ktL14GxulESj8cSTUCQowlBHUGhlHz58/view?usp=sharing

in case you're wondering why the first step in the second series of steps isn't showing an alert, it shouldn't be, from what I can see:
misleading_data

### CODE REVIEW

**Comment by andaleliz** [comment]
@youngfreezyVA I saw you added me as an assignee, but not sure exactly what is needed. I'm starting my leave today but will be around a _little_ bit this morning.  If we don't connect before I'm gone, perhaps Laurel or @rtwell can help? Ryan has more context around this than Laurel but I'm sure she'd be happy to help.

Also, we are 💯 in agreement RE the alert improvements. There is an issue out there ([#14485](https://github.com/department-of-veterans-affairs/va.gov-team/issues/14485)) that was generated from our mobile month initiative back in August.  It's a little tricky to determine exactly who to assign it to, because this a global problem. The design system team doesn't really provide design services so it falls to a team's product designers to pick it up. Unfortunately, most teams have specific initiatives/products they're focused on so global improvements don't get priority.

@jonathanjnelson on the Caregiver team had a need to incorporate an alert into an a design for a form he was working on, and agreed to submit his more mobile friendly alerts into the design system queue for consideration for the design system. Hopefully that will translate to a sitewide improvement our team can use too!

**Comment by jenniferlee-dsva** [comment]
@youngfreezyVA - great idea about a mobile version of in-body alerts. Let's keep an eye on caregiver team's work in flight. Mobile friendly alert would be a nice thig to incorporate in the future. 


Is there a way I can validate this?  I cannot access you local host link, nor can Gov's access Google doc links. Do you have a staging link @rtwell and I can look at?

File: src/site/layouts/step_by_step.drupal.liquid [hunk 1]
```diff
@@ -36,7 +36,7 @@
             {% endif %}
 
             <!-- Buttons -->
-            <div class="vads-u-margin-top--3">
+            <div class="vads-u-margin-top--3 vads-u-display--flex vads-u-justify-content--center medium-screen:vads-u-display--block">
```
Comment (kelsonic) [inline src/site/layouts/step_by_step.drupal.liquid:39]: Nice! 🔥

**Review by kelsonic — APPROVED** [review]
Looks great! 🎉 Thanks for doing this!

**Review by U-DON — APPROVED** [review]

**Comment by youngfreezyVA** [comment]
> @youngfreezyVA - great idea about a mobile version of in-body alerts. Let's keep an eye on caregiver team's work in flight. Mobile friendly alert would be a nice thig to incorporate in the future.
> 
> Is there a way I can validate this? I cannot access you local host link, nor can Gov's access Google doc links. Do you have a staging link @rtwell and I can look at?

hi jen, I'm about to merge this in, and then will have a staging link that I can share shortly, will comment here when I have it.

**Comment by youngfreezyVA** [comment]
> @youngfreezyVA - great idea about a mobile version of in-body alerts. Let's keep an eye on caregiver team's work in flight. Mobile friendly alert would be a nice thig to incorporate in the future.
> 
> Is there a way I can validate this? I cannot access you local host link, nor can Gov's access Google doc links. Do you have a staging link @rtwell and I can look at?

Hi @jenniferlee-dsva I have a staging preview link to share, as the staging preview deployment just completed - here you go:
http://preview-staging.vfs.va.gov/preview?nodeId=6899

**Comment by jenniferlee-dsva** [comment]
@youngfreezyVA - thank you. The mobile show expander works great. Very minor thing - sorry - please update the Show **Details** text with sentence casing - `Show details` .   You couldn't have known this; the VA.gov styling is sentence casing when not a proper noun or brand label.

**Pull request merged**